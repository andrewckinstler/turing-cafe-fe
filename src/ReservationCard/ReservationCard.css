article {
  background-color: #FFF;
  margin: 15px;
  width: 17%;
  border: 5px teal solid;
  border-radius: 5px;
}

.delete {
  margin: 15px 0;
  background-color: teal;
  color: #FFF;
  width: 150px;
  height: 30px;
}