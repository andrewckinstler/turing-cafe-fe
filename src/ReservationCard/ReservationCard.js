import React from 'react';

export const ReservationCard = ({ id, name, date, time, number, deleteReservation }) => {
  return(
    <div>
      <h3>{name}</h3>
      <p>{date}</p>
      <p>{time}</p>
      <p>Number of guests: {number}</p>
      <button onClick={() => deleteReservation(id)}>Cancel reservation</button>
    </div>
  )
}