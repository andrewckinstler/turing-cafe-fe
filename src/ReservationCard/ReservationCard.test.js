import React from 'react';
import { shallow } from 'enzyme'
import { ReservationCard } from './ReservationCard';

it('should match a snapshot', () => {
  const wrapper = shallow(<ReservationCard />)
  expect(wrapper).toMatchSnapshot()
});
