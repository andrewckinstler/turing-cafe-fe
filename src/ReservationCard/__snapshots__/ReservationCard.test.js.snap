// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match a snapshot 1`] = `
<div>
  <h3 />
  <p />
  <p />
  <p>
    Number of guests: 
  </p>
</div>
`;
